LICENSE-binary

```diff
@@ -324,10 +324,6 @@ org.apache.kerby:kerby-xdr:1.0.1
 org.apache.kerby:token-provider:1.0.1
 org.apache.yetus:audience-annotations:0.5.0
 org.apache.zookeeper:zookeeper:3.4.13
-org.codehaus.jackson:jackson-core-asl:1.9.13
-org.codehaus.jackson:jackson-jaxrs:1.9.13
-org.codehaus.jackson:jackson-mapper-asl:1.9.13
-org.codehaus.jackson:jackson-xc:1.9.13
```

Comment:
Jersey 1.x has a Jackson1 dependency and the Hadoop binary release contains Jackson1 jars. Therefore the lines cannot be removed for now.